shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -132,16 +139,17 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val excludeTransferPax: Option[Int] => Option[Int] = maybePax =>
-    maybePax.map(_ - TranPax.getOrElse(0)) match {
-      case Some(nonNegative) if nonNegative >= 0 => Some(nonNegative)
-      case Some(_) => Some(0)
-      case None => None
+  val excludeTransferPax: Option[Passengers] => Option[Passengers] = maybePax =>
+    maybePax.map { p =>
+      p.actual.map(_ - p.transit.getOrElse(0)) match {
+        case Some(nonNegative) if nonNegative >= 0 => p
+        case Some(_) => p.copy(actual = Some(0), transit = Some(0))
+        case None => p.copy(actual = None)
+      }
     }
 
-
   val bestPcpPaxEstimate: TotalPaxSource = {
```

Comment:
Let's make a copy of this function to return the best `Passengers`, and then have this function use it and call `getPcpPax`